New blog post: 2022 and Kubewarden

flavio opened the pull request:
This fixes https://github.com/kubewarden/kubewarden.io/issues/164

flavio commented:
I'm a bit late with that, sorry about that!

@divya-mohan0209, @nunix, @robertsirc: if any of you is still around, could you take a quick look at that please? thanks! :pray:

divya-mohan0209 commented on content/blog/2022/12/kubewarden-during-2022.md:
```diff
@@ -0,0 +1,108 @@
+
+By being part of CNCF, Kubewarden got also access to another CNCF project:
+[CLOMonitor](https://clomonitor.io/projects/cncf/kubewarden).
+Thanks to CLOMonitor, Kubewarden's users can have a clear picture of the health
```
```suggestion
Thanks to CLOMonitor, Kubewarden's users have a clear picture of the project health
```

divya-mohan0209 replied:
```suggestion
Thanks to CLOMonitor, Kubewarden's users have a clear picture of the project health
```

divya-mohan0209 commented on content/blog/2022/12/kubewarden-during-2022.md:
```diff
@@ -0,0 +1,108 @@
+By being part of CNCF, Kubewarden got also access to another CNCF project:
+[CLOMonitor](https://clomonitor.io/projects/cncf/kubewarden).
+Thanks to CLOMonitor, Kubewarden's users can have a clear picture of the health
+of the project and assess its development and security practices.
```
```suggestion
and can assess its development and security practices.
```

divya-mohan0209 commented on content/blog/2022/12/kubewarden-during-2022.md:
```diff
@@ -0,0 +1,108 @@
+By being part of CNCF, Kubewarden got also access to another CNCF project:
+[CLOMonitor](https://clomonitor.io/projects/cncf/kubewarden).
+Thanks to CLOMonitor, Kubewarden's users can have a clear picture of the health
+of the project and assess its development and security practices.
```
```suggestion
and can assess its development and security practices.
```

flavio replied:
I think there was some hiccup with GH: this suggestion was repeated a couple of times. I've accepted the 1st one

divya-mohan0209 commented on content/blog/2022/12/kubewarden-during-2022.md:
```diff
@@ -0,0 +1,108 @@
+By being part of CNCF, Kubewarden got also access to another CNCF project:
+[CLOMonitor](https://clomonitor.io/projects/cncf/kubewarden).
+Thanks to CLOMonitor, Kubewarden's users can have a clear picture of the health
+of the project and assess its development and security practices.
```
```suggestion
and can assess its development and security practices.
```

flavio replied:
I think there was some hiccup with GH: this suggestion was repeated a couple of times. I've accepted the 1st one

divya-mohan0209 commented on content/blog/2022/12/kubewarden-during-2022.md:
```diff
@@ -0,0 +1,108 @@
+During the month of August, Kubernetes 1.25 release finally removed
+[Pod Security Policies](https://kubernetes.io/docs/concepts/security/pod-security-policy/).
+
+This event was planned and advertised since a long time. Due to that,
```
```suggestion
This event was planned and advertised for a long time. Due to this,
```

divya-mohan0209 commented on content/blog/2022/12/kubewarden-during-2022.md:
```diff
@@ -0,0 +1,108 @@
+
+This event was planned and advertised since a long time. Due to that,
+the Kubewarden developers were able to provide
+[tooling, policies and documentation](https://docs.kubewarden.io/tasksDir/psp-migration)
```
```suggestion
[tooling, policies, and documentation](https://docs.kubewarden.io/tasksDir/psp-migration)
```

divya-mohan0209 commented on content/blog/2022/12/kubewarden-during-2022.md:
```diff
@@ -0,0 +1,108 @@
+This event was planned and advertised since a long time. Due to that,
+the Kubewarden developers were able to provide
+[tooling, policies and documentation](https://docs.kubewarden.io/tasksDir/psp-migration)
+to smooth the migration process from Kubernetes Pod Security Policies to
```
```suggestion
to smoothen the migration process from Kubernetes Pod Security Policies to
```

divya-mohan0209 commented on content/blog/2022/12/kubewarden-during-2022.md:
```diff
@@ -0,0 +1,108 @@
+
+The [sigstore](https://sigstore.dev) project proved to be one of the pillars of
+these initiatives.
+Kubewarden invested early on Sigstore, with some of our maintainers being
```
```suggestion
Kubewarden invested early in the Sigstore project, with some of our maintainers being
```

divya-mohan0209 reviewed (commented):
Initial set of reviews. Will leave more comments later.

flavio commented on content/blog/2022/12/kubewarden-during-2022.md:
```diff
@@ -0,0 +1,108 @@
+---
+title: Another year of Kubewarden
```
Good point, I've changed that

flavio commented on content/blog/2022/12/kubewarden-during-2022.md:
```diff
@@ -0,0 +1,108 @@
+With 2022 being basically over, it's time to look back at what happened within
+the Kubewarden project during the last twelve months.
+
+## The 1.0 release, CNCF inclusion
```
Done

flavio commented on content/blog/2022/12/kubewarden-during-2022.md:
```diff
@@ -0,0 +1,108 @@
+for the all Kubewarden's users who can now find all the existing policies
+on the [familiar ArtifactHub interface](https://artifacthub.io/packages/search?kind=13&sort=relevance&page=1).
+
+By being part of CNCF, Kubewarden got also access to another CNCF project:
```
I think there was some hiccup with GH: this suggestion was repeated a couple of times. I've accepted the 1st one

flavio commented on content/blog/2022/12/kubewarden-during-2022.md:
```diff
@@ -0,0 +1,108 @@
+for the all Kubewarden's users who can now find all the existing policies
+on the [familiar ArtifactHub interface](https://artifacthub.io/packages/search?kind=13&sort=relevance&page=1).
+
+By being part of CNCF, Kubewarden got also access to another CNCF project:
```
I think there was some hiccup with GH: this suggestion was repeated a couple of times. I've accepted the 1st one

flavio commented:
@divya-mohan0209 thanks for your suggestions. I've applied all of them, plus I've changed the title of the blog post and I've split the section about "v1.0 release + CNCF inclusion" into two dedicated units.

Do you want to give a final look before I merge it?

divya-mohan0209 commented on content/blog/2022/12/kubewarden-2022-wrapped.md:
```diff
@@ -0,0 +1,119 @@
+
+Next, we introduced a new operating mode for all our policies: the
+["monitor"](/blog/2022/05/monitor-mode/) mode. This allows Kubernetes administrators
+to better understand how a policy would impact their existing clusters once being
```
```suggestion
to better understand how a policy would impact their existing clusters once it was
```

divya-mohan0209 commented on content/blog/2022/12/kubewarden-2022-wrapped.md:
```diff
@@ -0,0 +1,119 @@
+enforced.
+
+More recently, the Kubewarden developers simplified the process of deploying
+Kubewarden inside of [air gapped](/blog/2022/11/airgap/) environments.
```
Do we want to provide a line or two on how we did this? It seems a bit abrupt.
```suggestion
Kubewarden inside of [air gapped](/blog/2022/11/airgap/) environments.
```

flavio replied:
I fear the article is already pretty long. The plan is to have the reader take a look at the linked blog post to learn more about that.

If you are against that, I can add a few lines like: 

```md
More recently, the Kubewarden developers simplified the process of deploying
Kubewarden inside of [air gapped](/blog/2022/11/airgap/) environments. This
has been achieved by providing small helpers, writing documentation and, most
important of all, dedicated QA efforts.
```

divya-mohan0209 replied:
That one line is enough IMO. The last line ended very abruptly which is why it felt like there needed to be something.

flavio replied:
Added

divya-mohan0209 commented on content/blog/2022/12/kubewarden-2022-wrapped.md:
```diff
@@ -0,0 +1,119 @@
+
+## New policies
+
+During the last months we created many ready to use policies.
```
```suggestion
During the last months we created many ready-to-use policies.
```

divya-mohan0209 commented on content/blog/2022/12/kubewarden-2022-wrapped.md:
```diff
@@ -0,0 +1,119 @@
+## New policies
+
+During the last months we created many ready to use policies.
+My favorite ones are:
```
Since this is a project blog, I think we should refrain from the usage of first-person pronouns like My/I and should refer to the collective. This should, normally, be the case unless we're publishing a case study by a user.
```suggestion
Some of them are:
```

divya-mohan0209 commented on content/blog/2022/12/kubewarden-2022-wrapped.md:
```diff
@@ -0,0 +1,119 @@
+
+## See you in 2023
+
+I think one of the major changes of 2023 will be the introduction of
```
```suggestion
One of the major changes in 2023 will be the introduction of
```

divya-mohan0209 commented on content/blog/2022/12/kubewarden-2022-wrapped.md:
```diff
@@ -0,0 +1,119 @@
+
+I think one of the major changes of 2023 will be the introduction of
+[community meetings](/blog/2022/12/community-meeting/).
+The first meeting is going to take place on January 12th, 2023 at 4 PM UTC.
```
This is maybe an overkill and you can choose to ignore this suggestion. But we should try being as timezone-agnostic as possible and include a hyperlink to a world clock that converts the time into different timezones. An example would be: https://www.worldtimebuddy.com/.
```suggestion
The first meeting will be held on January 12th, 2023 at 4 PM UTC.
```

divya-mohan0209 commented on content/blog/2022/12/kubewarden-2022-wrapped.md:
```diff
@@ -0,0 +1,119 @@
+[community meetings](/blog/2022/12/community-meeting/).
+The first meeting is going to take place on January 12th, 2023 at 4 PM UTC.
+
+We look forward to have you join these meetings to get in touch with the whole
```
```suggestion
We look forward to having you join these meetings and interacting with the rest of the
```

divya-mohan0209 commented on content/blog/2022/12/kubewarden-2022-wrapped.md:
```diff
@@ -0,0 +1,119 @@
+The first meeting is going to take place on January 12th, 2023 at 4 PM UTC.
+
+We look forward to have you join these meetings to get in touch with the whole
+Kubewarden community. Not only you will learn what is happening inside
```
```suggestion
Kubewarden community. Not only will you learn about the latest happenings within
```

divya-mohan0209 commented on content/blog/2022/12/kubewarden-2022-wrapped.md:
```diff
@@ -0,0 +1,119 @@
+
+We look forward to have you join these meetings to get in touch with the whole
+Kubewarden community. Not only you will learn what is happening inside
+of the project, but you will be able to share your feedback and Kubewarden stories
```
```suggestion
the project, but you will also be able to share your feedback and Kubewarden stories
```

divya-mohan0209 commented on content/blog/2022/12/kubewarden-2022-wrapped.md:
```diff
@@ -0,0 +1,119 @@
+of the project, but you will be able to share your feedback and Kubewarden stories
+with everybody else.
+
+In the meantime, on behalf of the Kubewarden developers, let me wish you a Happy New Year! 🍾 🎆 🎉
```
```suggestion
In the meantime, on behalf of the Kubewarden project, I'd like to wish all our users a Happy New Year! 🍾 🎆 🎉
```

divya-mohan0209 reviewed (commented):
Second set of reviews. Hope you find them useful!

divya-mohan0209 commented on content/blog/2022/12/kubewarden-2022-wrapped.md:
```diff
@@ -0,0 +1,120 @@
+acceptance into CNCF led to an exciting integration with
+[ArtifactHub](/blog/2022/07/artifact-hub-supports-kubewarden/).
+This proved to be a great improvement not just for policy authors, but also
+for the all Kubewarden's users who can now find all the existing policies
```
Nit.
```suggestion
for Kubewarden users who can now find all the existing policies
```

divya-mohan0209 commented on content/blog/2022/12/kubewarden-2022-wrapped.md:
```diff
@@ -0,0 +1,120 @@
+Sigstore provides different ways to sign artifacts: with
+traditional keys,
+[keyless mode](/blog/2022/05/verifying-image-signatures/),
+[keyless mode from a GitHub Action workflow](/blog/2022/07/verify-signatures-with-gha-and-prefix/) and
```
Not.
```suggestion
[keyless mode from a GitHub Action workflow](/blog/2022/07/verify-signatures-with-gha-and-prefix/) or
```

divya-mohan0209 commented on content/blog/2022/12/kubewarden-2022-wrapped.md:
```diff
@@ -0,0 +1,120 @@
+a [ready to use policy](https://artifacthub.io/packages/kubewarden/verify-image-signatures/verify-image-signatures)
+that can be used to vet the container images used by Kubernetes workloads.
+
+Sigstore provides different ways to sign artifacts: with
```
I think we should say the sigstore integration here instead of just Sigstore.
```suggestion
The Sigstore integration provides Kubewarden users different ways to sign artifacts: with
```

divya-mohan0209 commented on content/blog/2022/12/kubewarden-2022-wrapped.md:
```diff
@@ -0,0 +1,120 @@
+traditional keys,
+[keyless mode](/blog/2022/05/verifying-image-signatures/),
+[keyless mode from a GitHub Action workflow](/blog/2022/07/verify-signatures-with-gha-and-prefix/) and
+[self-signed certificates](/blog/2022/12/sigstore-certificate-verification/)
```
```suggestion
by using [self-signed certificates](/blog/2022/12/sigstore-certificate-verification/)
```

divya-mohan0209 commented on content/blog/2022/12/kubewarden-2022-wrapped.md:
```diff
@@ -0,0 +1,120 @@
+* [Deprecated resources](/blog/2022/11/deprecation-policy/): this policy can be
+  used to prevent the creation of resources that are deprecated by Kubernetes
+
+## See you in 2023
```
Non -blocking not.
```suggestion
## See you in 2023!
```

divya-mohan0209 reviewed (approved):
Mostly non-blocking nits. LGTM!

flavio commented:
Thanks @divya-mohan0209 for your help!
